### PR TITLE
Checkstyle for tests

### PR DESCRIPTION
**Pull request by kalaspuffar**
Hi @bertfrees and @PaulRambags

I've now gone through all the test code and fixed the checkstyle issues that occurred. Fixing these will remove a lot of the style issues we have encountered earlier. I will continue to do the same work on the src directory.

* Organize imports
* Format file
* Tabs to spaces
* Adding punctuations to comments so they could be used correctly in JavaDoc
* Renaming constants to UPPERCASE_UPPERCASE style
* Renaming member variables from constant form camelCase
* Fixing the length of strings to below 120 chars
* Changed println and printStacktrace statements to use a logger instead.
* Renamed variables, files, and directories that had _.

Best regards
Daniel

### CODE REVIEW

**Review by PaulRambags — APPROVED** [review]

**Comment by bertfrees** [comment]
> the checkstyle issues that occurred

Can you summarize which issues exactly? In cases like these where you can't see in the diff what has happened (too big), I think it's important to have a good description.

**Review by bertfrees — APPROVED** [review]
Thanks for the summary.

**Comment by kalaspuffar** [comment]
PR approved, Running regression tests.

**Comment by kalaspuffar** [comment]
Regression test ok, merging.
